docs/docs/behaviors/hold-tap.md

```diff
@@ -31,6 +31,12 @@ When the hold-tap key is released and the hold behavior has not been triggered,
 
 ![Hold-tap comparison](../assets/hold-tap/comparison.png)
 
+The 'hold-preferred' flavor tends to work well in combination with the shift, ctrl, caps lock, escape, tab and similar keys because they do not typically occur in the middle words or other rapid typing sequences in which the user may (unknowningly) execute a rolling keystrike, where they press the next key before the fully releasing the prior (hold/mod/layer-tap) key. Futhermore, because the decision to emit the hold behavior is made upon the second key being pressed, the 'hold-preferred' flavor most closely mimics the behavior of the shift key on a normal keyboard, and therefore tends to feel quite natural to users. However, because it biases towards the hold behavior in rolling keystrike situations, this flavor tends not to work well in combination with the alphas or for the spacebar, except for the small percentage of users that have very strict key-up habits.
```

Comment:
```suggestion
The 'hold-preferred' flavor tends to work well in combination with the shift, control, caps lock, escape, tab and similar keys because they do not typically occur in the middle words or other rapid typing sequences in which the user may (unknowingly) execute a rolling keystroke, where they press the next key before fully releasing the prior (hold/mod/layer-tap) key. Futhermore, because the decision to emit the hold behavior is made upon the second key being pressed, the 'hold-preferred' flavor most closely mimics the behavior of the shift key on a normal keyboard, and therefore tends to feel quite natural to users. However, because it biases towards the hold behavior in rolling keystroke situations, this flavor tends not to work well in combination with the alphas or the spacebar, except for the small percentage of users that have very strict key-up habits.
```
keystrike?

`alphas`?  As in alphanumeric keys`?

Reply:
I'll adopt keystroke

I'll replace "alphas" with "alphabetical characters." I wanted to avoid the term alphanumeric because IMO, the number keys are likely better (though still not ideally) suited for hold-preferred behaviors than the alphabetical characters, due to being slightly less frequent, and users being generally slightly less proficient/fast/shortcut-prone with them.

Reply:
Ah ok!  Or just ... letter keys?

Reply:
hah! yes, you're right, that's much clearer. Words... they hard sometimes